python/tvm/target/source_module.py

```diff
@@ -0,0 +1,66 @@
+# License .to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=no-else-return, unidiomatic-typecheck, undefined-variable, invalid-name, redefined-builtin
+"""
+Helper functions and classes for handling source and metdata.
+"""
+from tvm.runtime import _ffi_api
+
+class SourceMetadataModule:
+    """The module used to wrap both source and metadata."""
```

Comment:
Let us not expose it to the python side for now, as it is totally fine to get things from the mod itself. The only reason why have the graph runtime wrapper is that it is too commonly used